src/WatchJuliaBurn.jl

```diff
@@ -1,7 +1,31 @@
 module WatchJuliaBurn
 
-export c╯°□°ↄ╯
+export c╯°□°ↄ╯, 🗺, 📖
+export ✅, 👍, 👌, 👎
+export 😻, ⬇️😻, ➡️😻, 😹, ⬇️😹, ➡️😹, 🐈, ⬇️🐈, ➡️🐈
+export ㅠ
 
+## Base
 const c╯°□°ↄ╯ = throw
+const 🗺 = map
+const 📖 = Dict
+const ✅ = true
+const 👍 = true
+const 👌 = true
+const 👎 = false
+
+## Arrays
+const 😻 = cat
+const ⬇️😻 = vcat
+const ➡️😻 = hcat
+const 😹 = cat
+const ⬇️😹 = vcat
+const ➡️😹 = hcat
+const 🐈 = cat
+const ⬇️🐈 = vcat
+const ➡️🐈 = hcat
+
+## Math
+const ㅠ = π
```

Comment:
I am not sure this one is really needed :laughing:

Reply:
But why not!

Reply:
Exactly!